### PR TITLE
Malf AI in nearly any mode

### PR DESCRIPTION
**Pull request by ghost**
Malf AI can now happen in nearly any mode, 15% chance for most non-blob roundtypes. Configurable!

:cl: Thefastfoodguy
add: Malf AI can now happen in nearly any gamemode
/:cl:

yes i tested it. MALF_CHANCE in the hippie config is in %. Also, the chance will not roll if: The AI is jobbanned from: all antags, traitor, or malf, or has malf ai disabled in prefs

also a min pop for malf can be set (default at 20)

### CODE REVIEW

File: hippiestation/code/modules/jobs/job_types/silicon.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+/datum/job/ai/proc/can_be_malf(mob/living/M)
+	if(jobban_isbanned(M, "Syndicate") || jobban_isbanned(M, ROLE_MALF) || jobban_isbanned(M, ROLE_TRAITOR))
+		return FALSE
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/jobs/job_types/silicon.dm]: the sadist in me wants to add a little message here that says "You rolled a malf round!! But you got antag banned so no malf round for you... try following the rules in the future."

**Comment by PopNotes** [comment]
I know it seems minor, but I'd rather have the chance lowered to 10%. Even that is WAY higher odds for AI to roll malf than with the way it works currently.

**Comment by ghost** [comment]
malf does better to kill stragglers after the main crew is dead from other antags rather than close the round itself

i don't think adding it as a mulligan mode thing is a good idea

**Comment by Pyko1** [comment]
Would rather see malf as an rare event witn no alert that has higher weight to happen around 10-30 mins in to a round.

**Comment by DinduStuffin** [comment]
-1
I don't think we should be doing a reverse blob/abductor thing where we take an established mode and make it into a random event that can happen in every mode. It'll probably just lead to our cancer validhunters checking the AI nonstop every single round and being pedantic about everything it does.

**Comment by ghost** [comment]
@DinduStuffin i would like to remind you, it's a low, _configurable_ chance

**Comment by DinduStuffin** [comment]
it's a chance regardless. High or low, when it happens it's going to be ANOTHER thing for the crew to have to deal with, something for the antags to take advantage of in the crew's focus on the rogue AI. I think it's a terrible idea, and would basically lead to more murderboning.

**Comment by Super3222** [comment]
https://github.com/yogstation13/yogstation/pull/165

**Comment by PopNotes** [comment]
lol yogs

**Comment by Mlgdatboiwew** [comment]
+1 Like someone else said they chances should be lowered so you dont have plasma flooding every round

**Comment by j-mie** [comment]
would rather this was just in as a ION storm which wipes the AI laws and makes it malf as part of the post

**Pull request closed without merging**